### DISCUSSION TITLE
Bluetooth microphone not compatible

### DISCUSSION THREAD

**Discussion by bnrabander** [jamulussoftware/jamulus#1364] [General]
I have a Pioneer bluetooth headphone (SE-MS9BN). Jamulus reports that the headphone microphone is not compatible. Why is that?
MacBook Pro (15-inch, 2019)
MacOS Catalina 10.15.7

**Comment by hoffie** [comment]
Which message do you get?
If it is about the sample rate, you could try to change it (somewhere in your Mac's audio settings? Don't know detailsm, sorry).
Some devices cannot be changed apparently, but there are workarounds as described here: https://github.com/jamulussoftware/jamulus/discussions/1332#discussioncomment-527263

**Reply by dcorson-ticino-com** [reply]
Note that a bluetooth headphone adds quite a bit of delay to the sound and although it should work, it is not reccomended.
You will be happier using a cabled headphone for Jamulus.

**Comment by bnrabander** [comment]
Het geselecteerde audioapparaat kon niet worden gebruikt vanwege de volgende fout: Een sample rate van 16000 Hz voor het audio-ingangsapparaat van het huidige systeem wordt niet ondersteund. Open de Audio-MIDI-Setup in Applications->Utilities en probeer een sample rate van 48000 Hz in te stellen. Het vorige stuurprogramma zal worden geselecteerd.
Translated:
The selected audio devixce could not be used because of the following error. A sample rate of 16000 Hz for the audio input device of the current system is not supported etc.
And I cannot change the sample rate.
thanks for your answer
best, cb

**Reply by ann0see** [reply]
Yes. Probably you're out of luck then.

Please keep in mind that Bluetooth headphones are not usable with Jamulus. Getting a cheap microphone/headphone should be possible (maybe you already have one? Check your phone's headphones)
